.azure/gpu-tests-lite.yml

```diff
@@ -29,9 +30,8 @@ pr:
       - "src/lightning_lite/**"
       - "tests/tests_lite/**"
       - "setup.cfg"  # includes pytest config
-      - ".actions/**"
```

Comment:
You probably want to add setup.py here for consistency too

Reply:
no, you just requested to revert it :)